contracts/Rollup.sol

```diff
@@ -406,19 +397,34 @@ contract Rollup is RollupHelpers {
      * @notice finalise deposits and submit batch
      */
     function finaliseDepositsAndSubmitBatch(
-        uint256 _subTreeDepth,
-        Types.StateMerkleProofWithPath calldata zero
-    ) external payable onlyCoordinator isNotRollingBack {
-        bytes32 depositSubTreeRoot = depositManager.finaliseDeposits(
-            _subTreeDepth,
-            zero,
-            getLatestBalanceTreeRoot()
+        Types.CommitmentInclusionProof memory previous,
+        Types.SubtreeVacancyProof memory vacant
+    ) public payable onlyCoordinator isNotRollingBack {
```

Comment:
Solidity complains `Umimplemented Error` when we use calldata structs here, so I change the visibility to `public` here. 
In contrast to the case of submit transfer batch, where we want to send as much data and be gas efficient there, the submit deposit batch doesn't have the same requirement here. wdyt @vaibhavchellani

Reply:
Yeah I think that makes sense.